src/meo.py

```diff
@@ -73,7 +74,8 @@ def generate_inputs(data, filename_map):
 
         # TODO need to support partially directed graphs
         # Expected columns are Node1 EdgeType Node2 Weight
-        edges = data.get_interactome()
+        edges = data.get_undirected_interactome()
+        
```

Comment:
`meo` still had trouble with the input files (especially the interactome) because bidirected edges were represented as `(u,v)` and `(v,u)`.  I have made a `get_undirected_interactome()` function in `Dataset.py` that removes these `(u,v)` and `(v,u)` cases.

Reply:
I also agree with your suggestion above.  If MEO doesn't support bidirected edges represented as `(u,v)` and `(v,u)` (and other algorithms eventually do), its prepare inputs should be responsible for converting that edge pair into an undirected edge.